tests/spring-boot/pom.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>hawtio-tests</artifactId>
+    <groupId>io.hawt.tests</groupId>
+    <version>2.18-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>hawtio-tests-springboot</artifactId>
+  <name>${project.artifactId}</name>
+  <description>hawtio :: Spring Boot 2.x tests</description>
+  <packaging>jar</packaging>
+
+  <properties>
+    <!-- Jetty version referred from the spring-boot-dependencies project -->
+    <jetty.version>9.4.11.v20180605</jetty.version>
+    <hawtio.url>/actuator/hawtio</hawtio.url>
+    <hawtio.managementPort>10001</hawtio.managementPort>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${spring-boot-version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.camel.springboot</groupId>
+        <artifactId>camel-spring-boot-bom</artifactId>
+        <version>${camel-version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
```

Comment:
Do we need this? As we are in the same project now and don't need to use BOM for aligning versions. You can just use `project.version` for hawtio dependencies below.

Reply:
Yes, agree
I will remove hawtio-bom from springboot/pom dependencies